zbx_redis_stats.py

```diff
@@ -97,9 +97,8 @@ def llen():
                 print(client.llen(args.db))
 
             def llensum():
-                keys = client.keys('*')
```

Comment:
no it won't call it in each loop
scan_iter returns an iterator that a for-in statement can iterate through